lib/Sema/CSApply.cpp

```diff
@@ -7925,7 +7970,8 @@ Expr *TypeChecker::callWitness(Expr *base, DeclContext *dc,
 }
 
 Expr *
-Solution::convertBooleanTypeToBuiltinI1(Expr *expr, ConstraintLocator *locator) const {
+Solution::convertBooleanTypeToBuiltinI1(Expr *expr,
+                                        ConstraintLocator *locator) const {
```

Comment:
It would be nicer if this and other formatting changes were in a separate patch

Reply:
Indeed.  I split out the other general changes, but one off changes like this I left in.  They aren't systematic enough to cause problems for review I think, but I understand what you mean.